content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -212,6 +212,98 @@ Then set the Agent `dogstatsd_so_rcvbuf` configuration option to the same number
 dogstatsd_so_rcvbuf: 26214400
 ```
 
+## Client side telemetry
+
+Dogstatsd clients send telemetry metrics by default to the agent. This allows
+you to better troubleshoot where the bottleneck might be. Each metrics will be
+tag with the client language and the client version. Those metrics will not be
+counted as custom and will not be billed.
+
+Each client share a set of common tags.
+
+| Tag                | Description                                     | Example                |
+|--------------------|-------------------------------------------------|------------------------|
+| `client`           | The language of the client                      | `client:py`            |
+| `client_version`   | The version of the client                       | `client_version:1.2.3` |
+| `client_transport` | The transport byte the client (`udp` or `uds`)  | `client_transport:uds` |
+
+Keep in mind that when using UDP, network errors can't be detected by the client
+and the corresponding metrics will not reflect bytes/packets drop.
+
+{{< tabs >}}
+{{% tab "Python" %}}
+
+Starting with version `0.34.0` of the Python client.
+
+| Metrics Name                               | Metric Type | Description                                                                               |
+|--------------------------------------------|-------------|-------------------------------------------------------------------------------------------|
+| `datadog.dogstatsd.client.metrics`         | count       | Number of `metrics` sent to the client by yout application (before sampling).             |
+| `datadog.dogstatsd.client.event`           | count       | Number of `events` sent to the client by yout application.                                |
```

Comment:
sanity-check: is the `event` missing a `s` in the metric name?

Reply:
not but it should ! I'll update the python and go PR. :ok_hand: